src/main/java/edu/ksu/canvas/model/assignment/Progress.java

```diff
@@ -0,0 +1,108 @@
+package edu.ksu.canvas.model.assignment;
+
+import edu.ksu.canvas.annotation.CanvasObject;
+
+import java.util.Date;
+
```

Comment:
Please add some class-level javadoc here. Especially since this is kind of an odd class, it should have some documentation about what it is.